frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -134,6 +134,8 @@ class TableItem extends React.Component<{
         let bundleInfos = item.bundles_spec.bundle_infos;
         let headerItems = item.header;
         let headerHtml = headerItems.map((item, index) => {
+            item = item === 'uuid[0:8]' ? 'uuid' : item;
+            item = item === 'summary[0:1024]' ? 'summary' : item;
```

Comment:
This wasn't happening before...would be not good to hard code this.  It seems like we should be displaying the field name, not how it's computed?

Reply:
I found the table column is also `uuid[0:8]` in a PR submitted half a year ago [2190](https://github.com/codalab/codalab-worksheets/pull/2190). And it seems like the field name is also`uuid[0:8]`, so that's why we display it: https://github.com/codalab/codalab-worksheets/blob/34863d39ae696c343d6addc41a47c0a87c533271/codalab/lib/worksheet_util.py#L577.

Reply:
Ah, I see, the `uuid[0:8]` was there all along.  The `summary[0:1024]` is new because of another change, which is the part that looks a bit ugly.  But maybe we can just leave things as is and close this PR in the interest of being clear and explicit.

Reply:
Got it!